### ISSUE TITLE
 AdMob 7.4.1 and AppLovin 6.6.0 not working on iOS - missing bitcode, archive failed

### ISSUE THREAD

**Issue by SuperUnityCoder** [googleads/googleads-mobile-unity#2594]
### [REQUIRED] Step 1: Describe your environment

  * Unity version: 2020.3.42f1
  * Google Mobile Ads Unity plugin version: 7.4.1
  * Platform: iOS
  * Platform OS version: macOS 12.5.1, Xcode 13.4.1
  * Any specific devices issue occurs on: N/A
  * Mediation ad networks used, and their versions: AppLovin 6.6.0

### [REQUIRED] Step 2: Describe the problem
AppLovin mediation has missing bitcode. Archive fails with error:
[09:25:11] :	 [Step 1/2] ld: '/Users/*/Library/Developer/Xcode/DerivedData/Unity-iPhone-bxeghwopqqysdsdtviwotlzocmgf/Build/Intermediates.noindex/ArchiveIntermediates/Unity-iPhone/BuildProductsPath/Release-iphoneos/XCFrameworkIntermediates/GoogleMobileAdsMediationAppLovin/AppLovinAdapter.framework/AppLovinAdapter(GADMAdapterAppLovinUtils.o)' does not contain bitcode. You must rebuild it with bitcode enabled (Xcode setting ENABLE_BITCODE), obtain an updated library from the vendor, or disable bitcode for this target. file '/Users/*/Library/Developer/Xcode/DerivedData/Unity-iPhone-bxeghwopqqysdsdtviwotlzocmgf/Build/Intermediates.noindex/ArchiveIntermediates/Unity-iPhone/BuildProductsPath/Release-iphoneos/XCFrameworkIntermediates/GoogleMobileAdsMediationAppLovin/AppLovinAdapter.framework/AppLovinAdapter' for architecture arm64
[09:25:11] :	 [Step 1/2] clang: error: linker command failed with exit code 1 (use -v to see invocation)
[09:25:11] :	 [Step 1/2] 
[09:25:11]W:	 [Step 1/2] ** ARCHIVE FAILED **

#### Steps to reproduce:

Just call archive from xcodebuild.

#### Relevant Code:

Xcodebuild archive command:
`/Applications/Xcode.app/Contents/Developer/usr/bin/xcodebuild -workspace Unity-iPhone.xcworkspace -configuration Release -scheme Unity-iPhone clean archive -archivePath Unity-iPhone-archive -destination generic/platform=iOS`

**Comment by NVentimiglia** [comment]
Thank you, we will look into this issue. b/275365493 for reference.

**Comment by NVentimiglia** [comment]
@SuperUnityCoder 

Both the GMA Plugin version 7.4.1 and AppLovin Unity mediation plugin version 6.6.0 (which supports the iOS adapter version 11.7.1.0, which is built against GMA SDK version 10.0.0) has removed bitcode support.

Removing Bitcode from the project's Build Settings ("Enable Bitcode" set to NO) should make the project build.

**Comment by SuperUnityCoder** [comment]
After bitcode removal I'm getting errors in unity-plugin-library.a which looks exactly like in this report:
https://github.com/googleads/googleads-mobile-unity/issues/2531
But there are also errors coming from UnityAds 3.6.0:
```
"Foundation.URL.path(percentEncoded: Swift.Bool) -> Swift.String", referenced from:
    (extension in UnityAds):__C.NSFileManager.deleteFile(at: Foundation.URL) throws -> () in UnityAds(UnityAds-arm64-master.o)
"_objc_msgSend$operation", referenced from:
    -[WebViewInvokerQueueDecorator callInvokerUsingBufferObject:] in UnityAds(UnityAds-arm64-master.o)
"_objc_msgSend$callInvokerUsingBufferObject:", referenced from:
    ___53-[WebViewInvokerQueueDecorator invokeBufferedObjects]_block_invoke in UnityAds(UnityAds-arm64-master.o)
"_objc_msgSend$clearBuffer", referenced from:
    ___48-[WebViewInvokerQueueDecorator sdkDidInitialize]_block_invoke in UnityAds(UnityAds-arm64-master.o)
    ___52-[WebViewInvokerQueueDecorator sdkInitializeFailed:]_block_invoke in UnityAds(UnityAds-arm64-master.o)
"_objc_msgSend$buffer", referenced from:
    ___60-[WebViewInvokerQueueDecorator putBufferObjectIntoTheQueue:]_block_invoke in UnityAds(UnityAds-arm64-master.o)
    -[WebViewInvokerQueueDecorator clearBuffer] in UnityAds(UnityAds-arm64-master.o)
"_objc_msgSend$setOperation:", referenced from:
    -[WebViewInvokerQueueDecorator createBufferObjectWithOperation:withCompletion:andErrorCompletion:] in UnityAds(UnityAds-arm64-master.o)
"_objc_msgSend$createBufferObjectWithOperation:withCompletion:andErrorCompletion:", referenced from:
    -[WebViewInvokerQueueDecorator saveOperationIntoBuffer:withCompletion:andErrorCompletion:] in UnityAds(UnityAds-arm64-master.o)
"_objc_msgSend$putBufferObjectIntoTheQueue:", referenced from:
    -[WebViewInvokerQueueDecorator saveOperationIntoBuffer:withCompletion:andErrorCompletion:] in UnityAds(UnityAds-arm64-master.o)
"_objc_msgSend$callInvokerWithOperation:withCompletion:andErrorCompletion:", referenced from:
    -[WebViewInvokerQueueDecorator invokeOperation:withCompletion:andErrorCompletion:] in UnityAds(UnityAds-arm64-master.o)
    -[WebViewInvokerQueueDecorator callInvokerUsingBufferObject:] in UnityAds(UnityAds-arm64-master.o)
"_objc_msgSend$evaluateJavaScript:completionHandler:", referenced from:
    +[WKWebView(UserAgent) getUserAgentSyncFallBack:] in UnityAds(UnityAds-arm64-master.o)
"_objc_msgSend$runUntilDate:", referenced from:
    +[WKWebView(UserAgent) getUserAgentSyncFallBack:] in UnityAds(UnityAds-arm64-master.o)
"_objc_msgSend$initializeWithGameID:testMode:completion:error:", referenced from:
    +[UnityServices newStart:testMode:delegate:] in UnityAds(UnityAds-arm64-master.o)
```

**Comment by NVentimiglia** [comment]
Fixed in the latest release (8.0.0).
